src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -96,8 +101,17 @@ public PersonBuilder withEmail(String email) {
         return this;
     }
 
+    /**
+     * Sets the {@code Birthday} of the {@code Person} that we are building.
+     */
+    public PersonBuilder withBirthday(String birthday) {
+        this.birthday = new Birthday(birthday);
```

Comment:
Use `DateUtil.fromDateInput(...)` to get back a `LocalDate`. Use that to create `Birthday`. This will not change the tests that were already written.